appinventor/appengine/war/index.html

```diff
@@ -28,6 +28,10 @@
       </ul>
     </div>
     <script language="javascript" src="ode/ode.nocache.js"></script>
+    <script type="text/javascript" src="tutorials/tutorial.js"></script>
```

Comment:
can all these js files be minimised and pushed into one file so it only needs a HTTP request?
even better would be to add them to the GWT generated file, but that would need more work.